command/providers_test.go

```diff
@@ -75,14 +76,10 @@ func TestProviders_noConfigs(t *testing.T) {
 }
 
 func TestProviders_modules(t *testing.T) {
-	cwd, err := os.Getwd()
-	if err != nil {
-		t.Fatalf("err: %s", err)
-	}
-	if err := os.Chdir(testFixturePath("providers/modules")); err != nil {
-		t.Fatalf("err: %s", err)
-	}
-	defer os.Chdir(cwd)
+	td := tempDir(t)
+	copy.CopyDir(testFixturePath("providers/modules"), td)
+	defer os.RemoveAll(td)
+	defer testChdir(t, td)()
```

Comment:
Fixing a bug which left dangling `.terraform` state when running tests.

Reply:
🎉 thank you! that had even been pointed out to me and i forgot 🤦